### DISCUSSION TITLE
colab notebook project error 404: NOT_FOUND

### DISCUSSION THREAD

**Discussion by thinkscriptfan** [vercel/community#2699] [Help]
### Summary

created a simple google colab notebook  with 2 cells , one code and one text. named hello.ipynb

print("Hello World!")
This is a simple Python program that prints "Hello World!" to the console.

in google colab , I saved it to my github using save to github , to a test repository.
in vercel I add new project by import  from this repository , with success. 
however, it comes up with 
404: NOT_FOUND
Code: NOT_FOUND
ID: sfo1::wg5t2-1685641952047-a73b221718f0

### Steps to Reproduce

created a simple google colab notebook  with 2 cells , one code and one text. named hello.ipynb

print("Hello World!")
This is a simple Python program that prints "Hello World!" to the console.

in google colab , I saved it to my github using save to github , to a test repository.
in vercel I add new project by import  from this repository , with success. 
however, it comes up with 
404: NOT_FOUND
Code: NOT_FOUND
ID: sfo1::wg5t2-1685641952047-a73b221718f0

**Comment by smaeda-ks** [comment]
@thinkscriptfan Hard to tell what's wrong without having a reproducible example:
https://vercel.com/guides/creating-a-minimal-reproducible-example

Happy to check if you can provide that. Is it working locally (i.e., `vercel dev`)?

Otherwise, we have some examples here:
https://github.com/vercel/examples/tree/main/python/hello-world
https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/python

Make sure you're adding your Python code under `/api` folder, and try adding the `vercel.json` file at the project root with the following content:

```
{
  "rewrites": [{ "source": "/(.*)", "destination": "/api" }]
}
```

**Reply by thinkscriptfan** [reply]
here is the code ( in text ) that google colab generated and saved to github  under  hello.ipynb ( colab notebook) . I then create a project  by import from github . 
the code is below:
{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "authorship_tag": "ABX9TyM3jrpvxOibKvQrsKQfziKs",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/thinkscriptfan/colab_hello_world/blob/main/hello.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "QK84Jisl8bgE",
        "outputId": "25e59580-c2f6-4d4c-b1ed-882b32302556"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Hello World!\n"
          ]
        }
      ],
      "source": [
        "\n",
        "print(\"Hello World!\")\n"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# This is a simple Python program that prints \"Hello World!\" to the console."
      ],
      "metadata": {
        "id": "ngi5KX6h8iVi"
      }
    }
  ]
}
